### PR TITLE
Objects Automated Test - LPS-146873, LPS-146874 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-146873
https://issues.liferay.com/browse/LPS-146874

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectWorkflow.testcase [hunk 1]
```diff
@@ -23,12 +23,53 @@ definition {
...
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectWorkflow.testcase:40]: Hi @andreefarias 

You can add the parameter `fieldBusinessType = "Text"` ? 

thanks

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias 

Please, check my comments and make the necessary changes.

Thanks

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**